### PR TITLE
TURN server testing helpers

### PR DESCRIPTION
**Pull request by pizkaz**
A (WIP) collection of TURN server debugging helper scripts to save people the PITA I went through while finding out why the hell it is (still) not working properly.

### CODE REVIEW

**Comment by pizkaz** [comment]
It does have a self-contained help message explaining just that.
But we could mention it in [docs/others/troubleshooting.md](https://github.com/workadventure/workadventure/blob/develop/docs/others/troubleshooting.md).

File: docs/others/troubleshooting.md [hunk 1]
```diff
@@ -57,3 +57,19 @@ To enable "debug" log level, add a `DEBUG` environment variable:
 ```dotenv
 DEBUG: *
 ```
+
+## STUN/TURN issues
+
+To verify that your STUN/TURN server is working properly, you can use
+[TrickleIce](https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/)
+
+If you have configured a statich auth secret for TURN, you can use the script
```
Comment (moufmouf) [inline docs/others/troubleshooting.md:66]:
```suggestion
If you have configured a static auth secret for TURN, you can use the script
```

File: docs/others/troubleshooting.md [hunk 2]
```diff
@@ -57,3 +57,19 @@ To enable "debug" log level, add a `DEBUG` environment variable:
...
+[compute_turn_credentials.sh](https://github.com/workadventure/workadventure/tree/develop/contrib/tools/compute_turn_credentials.sh)
```
Comment (moufmouf) [inline docs/others/troubleshooting.md:67]:
```suggestion
[`compute_turn_credentials.sh](https://github.com/workadventure/workadventure/tree/develop/contrib/tools/compute_turn_credentials.sh)
```

Comment (pizkaz) [reply docs/others/troubleshooting.md]: Done.

File: docs/others/troubleshooting.md (hunk 1 above)
Comment (pizkaz) [reply docs/others/troubleshooting.md]: thx! Fixed.

**Comment by moufmouf** [comment]
LGTM! If you want, you can remove the Draft status and I'll merge this.

**Pull request merged**